retux/client/mixins/respondable.py

```diff
@@ -0,0 +1,56 @@
+from ..bot import Bot
+from ...api.http import _Route, _RouteMethod
+from .editable import Editable
+
+from attrs import asdict
+
+__all__ = ["Respondable"]
+
+
+class Respondable(Editable):
+    """
+    A mixin for objects with send/edit/delete methods
+
+    Methods
+    _______
+    respond() : `dict`
+        Executes a `respond` action to discord with the given Route and keyword arguments.
+    send() : `dict`
+        An alias of `respond()`
+    """
+
+    async def respond(self, bot: Bot, path: str, **kwargs) -> dict:
+        """
+        Executes a `respond` action to discord with the given Route and keyword arguments.
```

Comment:
I get what's being meant here but I'm lost what the "respond action" is. Did you mean "Executes a `Create Interaction Response` action" ?

```suggestion
        Executes a `respond` action to Discord with the given route and keyword arguments.
```